Model/CreditCardPaymentMethod.php

```diff
@@ -148,6 +148,23 @@ private function _convertAmountCurrency($fAmount, $fFromFactor, $oToCurrency)
      */
     public function getConfigFields()
     {
+        $iUrlFieldsOffset = 1;
```

Comment:
I think this should be in a helper, e.g.:

```php
public static function insertToArrayAtPosition(&$aArray, $aContents, $iIndex)
{
    $aArray = array_merge(
        array_slice($aArray, 0, $iIndex, true),
        $aContents,
        array_slice($aArray, $iIndex, count($aArray) - 1, true)
    );
}
```

Reply:
:+1: